Controller/Rewrite/Adminhtml/System/Config/Save.php

```diff
@@ -3,9 +3,8 @@
  * EkomiIntegration Rewrite Config Save Controller
  *
  * @category    Ekomi
- * @package     Ekomi_EkomiIntegration
- * @author      Ekomi Private Limited
- *
+ * @copyright   Copyright (c) 2018 Ekomi ltd (http://www.ekomi.de)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
```

Comment:
@atif-coeus @SyedAbdullahShah Why OSL3.0, for an enterprise owned integration? Are you bounded to work open source for this project? 

I believe GNU GPLv3, suits well with the domain of the project.

Reply:
This is free plugin and anybody can extend/customise it according to his/her needs

Reply:
But it deepens on an enterprise tool which requires a commercial license. So to use this piece of software, one need to have a commercial license (or registration with Ekomi), so its not suitable for OSL3.0.

That's my understanding, OSL licenses are attributed to completely open source software where all dependencies also attributed to open source licenses.